File: Data Structures & Algorithms/binary-search/submission-0.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
        int n = nums.size();
        int s =0 ;
        int e = n-1;
        int mid = s + (e-s)/2;
        while (s<=e){
            mid = s + (e-s)/2;
            if(nums[mid]>target){
                e = mid-1;
            }
            else if(nums[mid]<target){
                s = mid+1;
            }
            else if(nums[mid]==target){
                return mid;
            }
        }
        return -1;
        
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answer = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if(nums[i]+nums[j]==target and i!=j):
                    answer.append(i)
                    answer.append(j)
        answer = set(answer)
        answer = list(answer)
        return answer

        